internal/clients/http/ec2/ec2_client.go

```diff
@@ -53,13 +54,11 @@ func newAssumedEC2ClientWithRegion(ctx context.Context, arn string, region strin
 	return &EC2Client{
 		ec2:     ec2.NewFromConfig(newCfg),
 		context: ctx,
-		log:     logger.With().Str("client", "ec2").Logger(),
+		logger:  logger,
```

Comment:
Why do you think it was wrang to enrich the logger by the client field? It might be useful to know, that the logs came from the client and not our app.

Reply:
Oh I moved this up, look it few lines up. If it's not there, then it's a bug, but I swear I did put it there.